setup.cfg

```diff
@@ -1,5 +1,13 @@
+[bumpversion]
+current_version = 0.0.1
```

Comment:
Sure agreed, but there are unimplemented features that I would argue need to be there for a complete first draft to alpha/beta with. Perhaps that is more conservative than necessary, but that's my opinion.

I'm not in any position to be dictating what the first version is here anyways since that still needs to be set up initially after this PR is merged, and that's not up to me haha. I just used 0.0.1 because that is what it was.